Tweaks/TweaksAssembly/BombStatus.cs

```diff
@@ -81,7 +81,7 @@ public void LateUpdate()
 		{
 			string formattedTime = currentBomb.GetFullFormattedTime;
 			TimerPrefab.text = formattedTime;
-			TimerShadowPrefab.text = formattedTime.Select(character => character == ':' ? character : '8').Join("");
+			TimerShadowPrefab.text = formattedTime.Select(character => character == ':' || character == '.' ? ':' : '8').Join("");
```

Comment:
Why did you change it from `character` to `':'`? Shouldn't the shadow text be a period as well?
 
 Also, could you use the `.EqualsAny` method to simplify the logic?

Reply:
1. The in-game bomb timer's shadow always displays `:`, I just thought it made sense.
2. Yes, I'm a dumbass.